internal/bucket/lifecycle/lifecycle_test.go

```diff
@@ -619,7 +629,7 @@ func TestHasActiveRules(t *testing.T) {
 				t.Fatalf("Got unexpected error: %v", err)
 			}
 			if got := lc.HasActiveRules(tc.prefix); got != tc.want {
-				t.Fatalf("Expected result with recursive set to false: `%v`, got: `%v`", tc.want, got)
+				t.Fatalf("Expected result: `%v`, got: `%v`", tc.want, got)
```

Comment:
Updated test error log to remove superfluous `recursive` parameter (from the past).